4-tools/spyderexample.py

```diff
@@ -1,28 +1,19 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
-Created on Mon Nov  9 18:03:24 2020
+Created on Thu Nov  4 15:39:32 2021
 
-@author: danielmikkola
+@author: Simon
 """
```

Comment:
None of this is useful.

Reply:
Sure. Spyder adds it automatically but I removed it now.